packages/headless/src/controllers/generated-answer/headless-generated-answer.test.ts

```diff
@@ -113,6 +124,18 @@ describe('generated answer', () => {
     expect(analyticsAction).toBeTruthy();
   });
 
+  it('#dislike dispatches no analytics action when #disliked is set to true', () => {
+    engine.state.generatedAnswer.disliked = true;
+    generatedAnswer.dislike();
+    const action = findAction(likeGeneratedAnswer.type);
+    const analyticsAction = engine.findAsyncAction(
+      logLikeGeneratedAnswer().pending
+    );
```

Comment:
I don't think this test is passing for the correct reason 😁 

```suggestion
    const action = findAction(dislikeGeneratedAnswer.type);
    const analyticsAction = engine.findAsyncAction(
      logDislikeGeneratedAnswer().pending
    );
```